<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>How a jumping gene shaped the human skin color evolution  - Hex Index</title>
  <link rel="stylesheet" href="../../styles.css">
</head>
<body class="reading-mode">
  <header class="reading-header">
    <a href="../../index.html" class="back-link">&larr; Back to Library</a>
  </header>
  <main class="reading-content">
    
    <article class="article-page">
      <header class="article-header">
        <h1>How a jumping gene shaped the human skin color evolution </h1>
        <div class="article-meta">
          <span class="author">By Various</span>
          <span class="separator">&middot;</span>
          <a href="../../publication/gwasstories/index.html" class="publication">
            GWAS Stories
          </a>
          <span class="separator">&middot;</span><time>Jul 28, 2024</time>
          <span class="separator">&middot;</span>
          <span class="read-time">10 min read</span>
        </div>
      </header>

      

      <div class="article-excerpt">
        <p>Regular readers of my Substack would recognize my special interest in noncoding genome. Today, we explore a fascinating tale of how a mobile genetic element has been coloring the evolutionary journey of our human ancestors for hundreds of thousands of years. </p><div class="captioned-image-container"><figure><a class="image-link image2 is-viewable-img" target="_blank" href="https://substackcdn.com/image/fetch/$s_!w4XY!,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F260c50e8-b1fa-4282-a4ee-807647139fdc_2206x1360.png" data-component-name="Image2ToDOM"><div class="image2-inset"><picture><source type="image/webp" srcset="https://substackcdn.com/image/fetch/$s_!w4XY!,w_424,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F260c50e8-b1fa-4282-a4ee-807647139fdc_2206x1360.png 424w, https://substackcdn.com/image/fetch/$s_!w4XY!,w_848,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F260c50e8-b1fa-4282-a4ee-807647139fdc_2206x1360.png 848w, https://substackcdn.com/image/fetch/$s_!w4XY!,w_1272,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F260c50e8-b1fa-4282-a4ee-807647139fdc_2206x1360.png 1272w, https://substackcdn.com/image/fetch/$s_!w4XY!,w_1456,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F260c50e8-b1fa-4282-a4ee-807647139fdc_2206x1360.png 1456w" sizes="100vw"><img src="https://substackcdn.com/image/fetch/$s_!w4XY!,w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F260c50e8-b1fa-4282-a4ee-807647139fdc_2206x1360.png" width="1456" height="898" data-attrs="{&quot;src&quot;:&quot;https://substack-post-media.s3.amazonaws.com/public/images/260c50e8-b1fa-4282-a4ee-807647139fdc_2206x1360.png&quot;,&quot;srcNoWatermark&quot;:null,&quot;fullscreen&quot;:null,&quot;imageSize&quot;:null,&quot;height&quot;:898,&quot;width&quot;:1456,&quot;resizeWidth&quot;:null,&quot;bytes&quot;:402983,&quot;alt&quot;:null,&quot;title&quot;:null,&quot;type&quot;:&quot;image/png&quot;,&quot;href&quot;:null,&quot;belowTheFold&quot;:false,&quot;topImage&quot;:true,&quot;internalRedirect&quot;:null,&quot;isProcessing&quot;:false,&quot;align&quot;:null,&quot;offset&quot;:false}" class="sizing-normal" alt="" srcset="https://substackcdn.com/image/fetch/$s_!w4XY!,w_424,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F260c50e8-b1fa-4282-a4ee-807647139fdc_2206x1360.png 424w, https://substackcdn.com/image/fetch/$s_!w4XY!,w_848,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F260c50e8-b1fa-4282-a4ee-807647139fdc_2206x1360.png 848w, https://substackcdn.com/image/fetch/$s_!w4XY!,w_1272,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F260c50e8-b1fa-4282-a4ee-807647139fdc_2206x1360.png 1272w, https://substackcdn.com/image/fetch/$s_!w4XY!,w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F260c50e8-b1fa-4282-a4ee-807647139fdc_2206x1360.png 1456w" sizes="100vw" fetchpriority="high"></picture><div class="image-link-expand"><div class="pencraft pc-display-flex pc-gap-8 pc-reset"></div></div></div></a><figcaption class="image-caption">Figure 5d from Kamitaki et al. <em>Nat Gen</em> 2024 illustrating the two mobile element insertions in to an intron of ASIP, encoding agouti signaling protein, during the human evolution. </figcaption></figure></div><p>One of the biggest mysteries that scientists were wrestling with during the early days was the puzzling disconnect between size of the organisms and their genomes. Larger animals have smaller genomes, whereas some plants, insects and even some unicellular organisms (e.g. Amoeba dubia) have humungous genomes. Just a couple of months ago, scientists found on the forest floors of an Australian island a plant species with the largest genome ever known to humans (<a href="https://doi.org/10.1016/j.isci.2024.109889">Fern√°ndez et al. </a><em><a href="https://doi.org/10.1016/j.isci.2024.109889">iScience</a></em><a href="https://doi.org/10.1016/j.isci.2024.109889"> 2024</a>). Tmesipteris oblanceolata, a tiny fern species endemic to eastern Australia has a genome that is 50 times larger than that of ours (<a href="https://www.nytimes.com/2024/05/31/science/largest-genome-fern-plant.html?unlocked_article_code=1.wE0.uHx6.SkXqQwyZUHBI&amp;smid=url-share&amp;utm_source=carlzimmer&amp;utm_medium=email&amp;utm_campaign=fridays-elk-life-keeps-throwing-more-junk-at-us">NYT article</a>). </p><p>The puzzling relationship between the genome size and organismal complexity, described as 'C-value paradox', is now explained by the fact that a major part of genomes are 'graveyards' of repetitive elements containing fossilized genomes of ancient viruses. More than 45% of the human genome are repeat elements filled with transposons, aka, jumping genes. Transposons jump from one part of the genome to another and, in the process, edit, delete or create new genes, bestowing upon its host new phenotypes. They are one of the major drivers of human evolution. </p><p>Just a month ago, there was a big breakthrough in genome editing where researchers from the Arc Institute in the US and University of Tokyo in Japan uncovered the molecular mechanism through which an evolutionarily old, most simplest forms of mobile elements called insertion sequences (IS) found in bacterial and archaeal species jump across the genome(s) (<a href="https://www.nature.com/articles/s41586-024-07552-4">Durrant et al. </a><em><a href="https://www.nature.com/articles/s41586-024-07552-4">Nature</a></em><a href="https://www.nature.com/articles/s41586-024-07552-4"> 2024</a>, <a href="https://www.nature.com/articles/s41586-024-07570-2">Hiraizumi et al. Nature 2024</a>). The breakthrough part is the realization that a component of this ancient transposition machinery was a noncoding RNA that bridges the transposon itself (bound to its one loop) with the target DNA integration site (bound to its other loop). Both the loops of this "bridge RNA" are programmable. So, by altering the RNA loop sequences, one can trick the recombinase enzyme that is part of the </p>...</source>
      </div>

      <div class="read-full-article">
        <a href="https://www.gwasstories.com/p/how-a-jumping-gene-shaped-the-human" class="read-button" target="_blank" rel="noopener">
          Read full article on GWAS Stories &rarr;
        </a>
        <p class="copyright-note">
          This excerpt is provided for preview purposes.
          Full article content is available on the original publication.
        </p>
      </div>
    </article>
  
  </main>
</body>
</html>